cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -813,27 +844,34 @@ public KafkaBrokerConfigurationBuilder withUserConfiguration(KafkaConfiguration
     }
 
     /**
-     * Add metric reporters to the corresponding Kafka configuration
+     * Adds the Kafka metric.reporters to the user configuration.
      *
-     * @param userConfig                The User configuration - Kafka broker configuration options specified by the user in the Kafka custom resource
-     * @param injectCcMetricsReporter   Inject the Cruise Control Metrics Reporter into the configuration
-     * @param injectJmxReporter         Inject the JMX Reporter into the configuration
-     */
-    private void addMetricReporters(KafkaConfiguration userConfig, boolean injectCcMetricsReporter, boolean injectJmxReporter) {
-        if (injectJmxReporter)  {
-            if (userConfig.getConfigOption("metric.reporters") != null && !userConfig.getConfigOption("metric.reporters").contains(KAFKA_JMX_REPORTER_CLASS)) {
-                userConfig.setConfigOption("metric.reporters", userConfig.getConfigOption("metric.reporters") + "," + KAFKA_JMX_REPORTER_CLASS);
-            } else {
-                userConfig.setConfigOption("metric.reporters", KAFKA_JMX_REPORTER_CLASS);
-            }
+     * @param userConfig The user configuration to which the metric reporters will be added.
+     * @param injectCcMetricsReporter Flag indicating whether to inject the Cruise Control Metrics Reporter.
+     * @param injectKafkaJmxReporter  Inject the JMX Reporter into the configuration
+     * @param injectStrimziMetricsReporter Flag indicating whether to inject the Strimzi Metrics Reporter.
+     */
+    private void maybeAddMetricReporters(KafkaConfiguration userConfig, boolean injectCcMetricsReporter, boolean injectKafkaJmxReporter, boolean injectStrimziMetricsReporter) {
+        if (injectCcMetricsReporter) {
+            createOrAddListConfig(userConfig, "metric.reporters", CruiseControlMetricsReporter.CRUISE_CONTROL_METRIC_REPORTER);
+        }
+        if (injectKafkaJmxReporter) {
+            createOrAddListConfig(userConfig, "metric.reporters", "org.apache.kafka.common.metrics.JmxReporter");
```

Comment:
What about having a constant for "org.apache.kafka.common.metrics.JmxReporter" within the `JmxPrometheusExporterModel` class?

Reply:
Yes fine, you already have my green check ;-)